pkg/test/exec.go

```diff
@@ -14,11 +15,16 @@ type Exec struct {
 }
 
 func (e *Exec) Run(path string) (string, error) {
-	src := e.Command
+	args := strings.Fields(e.Command)
 	if path != "" {
-		src = "PATH=" + path + ":$PATH " + src
+		abspath, err := filepath.Abs(path)
+		if err != nil {
+			return "", err
+		}
+		args[0] = filepath.Join(abspath, args[0])
 	}
-	cmd := exec.Command("/bin/bash", "-c", src)
+
+	cmd := exec.Command(args[0], args[1:]...)
```

Comment:
If you'd like to stay closer to the original behavior, you could do this.
```go
	if path != "" {
		savedpath := os.Getenv["PATH"]
		os.Setenv("PATH", path+os.PathListSeparator+savedpath)
		defer os.Setenv("PATH", savedpath)
	}
	args := strings.Fields(e.Command)
	cmd := exec.Command(args[0], args[1:]...)
```

Reply:
I kind of like using an explicit path in the arg over relying on $PATH. As mccanne might say "Chocolate vs Vanilla".

Reply:
👍